.circleci/config.yml

```diff
@@ -4,7 +4,15 @@ jobs:
   test:
     docker:
       - image: circleci/python:3.8
-      - image: circleci/postgres:12-alpine-ram
```

Comment:
FYI, apparently the `12-alpine-ram`  image is not up to date with the latest minor release (12.3). It's probably simpler to use `postgres:12-ram`, since the local development environment doesn't use the Alpine image.